service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -60,14 +60,15 @@ public JWTData getUserJWTData(String email) {
       Users user = maybeUser.get();
       Integer userId = user.getId();
 
-      Optional<UserTeam> mayberUserTeam =
+      Optional<UserTeam> mayberUserTeam = // loop into array of teams
           Optional.ofNullable(
               db.selectFrom(USER_TEAM)
```

Comment:
Do we need this anymore in JWT or can it be removed too? userTeam is unused on line 70.

Reply:
Can just `return new JWTData(user.getId(), user.getPrivilegeLevel()` probably